src/layer/VideoOverlay.js

```diff
@@ -49,7 +49,10 @@ export var VideoOverlay = ImageOverlay.extend({
 		// Fired when the video has finished loading the first frame
 		vid.onloadeddata = Util.bind(this.fire, this, 'load');
 
-		if (wasElementSupplied) { return; }
+		if (wasElementSupplied) {

```

Comment:
I'm not sure it's the right thing to do in case of `VideoOverlay` - `video`'s `src` attribute [is optional](https://www.w3schools.com/tags/tag_video.asp) and it may contain `source` elements instead.

Reply:
You are right it's not always display the actual video src. But, because VideoOverlay extends ImageOverlay, I think that this._url should keep also url and not element. I looked at the code of ImageOverlay and I noticed that only _initImage uses this value ,and _overlayOnError and setUrl change the value of this._url. I understand that this field is not used after the first call of _initImage.
So, I believe we should cancel the last commit, or if we stay it, to apply the changes for ImageOverlay and VideoOverlay

Reply:
You are right, I will fix it;